### PR TITLE
Support detach

### LINKED ISSUE #2: Add an options to allow disconnect process automatic.
eg: `deemon --detach xxxxx;`

### PR DESCRIPTION
**Pull request by Kingwl**
Fixes #2

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -166,12 +167,18 @@ async function main(command: Command, options: Options): Promise<void> {
 		console.log('Build daemon exited.');
 		process.exit(0);
 	});
+
+	if (options.detach) {
+		console.log('Build daemon detach.');
```
Comment (joaomoreno) [inline src/main.ts]:
```suggestion
		console.log('Detached from build daemon.');
```

**Review by joaomoreno — APPROVED** [review]
Thanks!

**Comment by joaomoreno** [comment]
Published 1.6.0

**Comment by Kingwl** [comment]
Thanks

**Pull request merged**